spec/fieldhand/paginator_spec.rb

```diff
@@ -170,8 +171,8 @@ module Fieldhand
 
       it 'sends an authorization header with a bearer token' do
```

Comment:
Do we need _both_ custom headers and a bearer token here or would just a bearer token suffice?